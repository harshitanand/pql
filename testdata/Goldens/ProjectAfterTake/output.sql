WITH "__subquery0" AS (SELECT * FROM "StormEvents" ORDER BY "EventId" ASC NULLS FIRST LIMIT 3)
SELECT "State" AS "State", "EventType" AS "EventType", "DamageProperty" AS "DamageProperty" FROM "__subquery0";
