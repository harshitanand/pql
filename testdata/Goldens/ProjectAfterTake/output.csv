State,EventType,DamageProperty
ATLANTIC SOUTH,Waterspout,0
FLORIDA,Heavy Rain,0
GEORGIA,Thunderstorm Wind,2000
